eng/Versions.props

```diff
@@ -27,7 +27,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Versions used by several individual references below -->
-    <RoslynDiagnosticsNugetPackageVersion>2.9.6</RoslynDiagnosticsNugetPackageVersion>
+    <RoslynDiagnosticsNugetPackageVersion>3.0.0-beta2.20059.3+77df2220</RoslynDiagnosticsNugetPackageVersion>
```

Comment:
@dotnet/roslyn-infrastructure @jaredpar  I've update the version number for roslyn analyzer package. I'm seeing the new analyzers kick in within VS, but it looks like `build.cmd` is still using some older version (2.9.6 as far as I can tell). 
Is that expected? What's the process from here?

Reply:
@jcouv - is command line build using older version of the analyzer package OR compiler toolset?

Reply:
@mavasani Not sure how to tell the difference. Probably both. 
Assuming that analyzer package is referenced by compiler toolset (correct me if that's wrong), there is no compiler toolset which references the new analyzer package that I need (since it's brand new).